src/sweeps/params.py

```diff
@@ -183,6 +190,14 @@ def ppf(self, x: ArrayLike) -> Any:
                     x, self.config["min"], self.config["max"] - self.config["min"]
                 )
             )
+        elif self.type == HyperParameter.INV_LOG_UNIFORM:
+            return 1.0 / np.exp(
+                stats.uniform.ppf(
+                    1 - x,
+                    -self.config["max"],
+                    self.config["max"] - self.config["min"],
+                )
+            )
```

Comment:
i think this should be

```suggestion
            return np.exp(
                -stats.uniform.ppf(
                    1 - x,
                    self.config["min"],
                    self.config["max"] - self.config["min"],
                )
            )
```